libbpf-tools/cpufreq.c

```diff
@@ -0,0 +1,239 @@
+// SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause)
+// Copyright (c) 2020 Wenbo Zhang
+//
+// Based on cpufreq(8) from BPF-Perf-Tools-Book by Brendan Gregg.
+// 10-OCT-2020   Wenbo Zhang   Created this.
+#include <argp.h>
+#include <signal.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include <linux/perf_event.h>
+#include <asm/unistd.h>
+#include <bpf/libbpf.h>
+#include <bpf/bpf.h>
+#include "cpufreq.h"
+#include "cpufreq.skel.h"
+#include "trace_helpers.h"
+
+#define FREQ	99
```

Comment:
oh, btw, this seems like a good candidate to be an optional command line argument, no?

Reply:
Yeah, agree, like perf -F :) I'll change it to arg.

Reply:
Fixed!